R/updateGate.R

```diff
@@ -0,0 +1,21 @@
+#' Update gate
+#'
+#' Updates a gate.
+#'
+#' @param experimentId ID of experiment.
+#' @param gateId Gate family ID
```

Comment:
Looks like this is just a typo because you got `updateGateFamily` correct, but this should just say `"Gate ID"`.

I just realized I didn't share the slide that explains tailoring and what gateIds vs GIDs are. Just did, sorry! https://docs.google.com/presentation/d/1azYlWyiPWELmTNOsU90SA4M3KaCx6mjVkQrZqnqOjwg/edit?usp=sharing Ping me in slack if you want to go through it. Looks like you mostly figured it out though 👏